sandbox.py

```diff
@@ -0,0 +1,30 @@
+from customersupport.models import Ticket, CallLog
+from customersupport.database import db_session
+from customersupport.database import init_db
+import datetime
+
+init_db()
+
+today = datetime.datetime.now()
+
+cl_1 = {"date_called":today,"calling_number":"111-222-3333","callback_number":"111-222-3434","notes":"Nothing to report here","employee":"1","ticket":1}
```

Comment:
I know we aren't enforcing PEP style, but this line is not very readable. Some spaces and newlines would help here.